citylearn/energy_model.py

```diff
@@ -913,6 +913,19 @@ def get_current_efficiency(self, energy: float) -> float:
 
         return efficiency
 
+    def set_ad_hoc_charge(self, energy: float):

```

Comment:
Why does the battery need to be charged this way after a transit state?

Reply:
Not sure, really, can recah out to Tiago.